### PR TITLE
Issue 105 post users route

### PR DESCRIPTION
**Pull request by almazhankapan**
checked post request with postman [here](https://slack-files.com/T01LN2C2L20-F01T9UDA4AG-f167a2f3f2)

### CODE REVIEW

File: back-end/app.js [hunk 1]
```diff
@@ -22,4 +24,19 @@ app.get('/tv_users/:id', (req, res, next) => {
         })
 });
 
+app.post('/tv_users', (req, res, next) => {
+    axios.post(`https://my.api.mockaroo.com/tv_users.json?key=${process.env.REACT_APP_MOCKAROO_KEY}&__method=POST`, {
```
Comment (LevBernstein) [inline back-end/app.js:28]: Please switch to MOCKAROO_KEY, instead of REACT_APP_MOCKAROO_KEY, thanks.

**Review by LevBernstein — CHANGES_REQUESTED** [review]

File: back-end/app.js (hunk 1 above)
Comment (almazhankapan) [reply back-end/app.js]: yes, just did

**Comment by almazhankapan** [comment]
tested post request to tv_users and patch request to tv_users/id with postman
[here](https://slack-files.com/T01LN2C2L20-F01T9UDA4AG-f167a2f3f2) and [here](https://slack-files.com/T01LN2C2L20-F01SX0NMQTY-12e9019075)

**Review by sam-eng — APPROVED** [review]
Looks good to me!

**Comment by almazhankapan** [comment]
> Looks good to me!

thank you :)

**Pull request merged**